Directory.Packages.props

```diff
@@ -1,12 +1,16 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-  </PropertyGroup>
-
-  <PropertyGroup>
     <OTelLatestStableVer>1.6.0</OTelLatestStableVer>
   </PropertyGroup>
 
+  <Target Name="PreparePackageValidation" BeforeTargets="Restore">
+    <PropertyGroup>
+      <EnablePackageValidation Condition="'$(EnablePackageValidation)' == ''">true</EnablePackageValidation>
+      <PackageValidationBaselineVersion Condition="'$(MinVerTagPrefix)' == 'core-'">$(OTelLatestStableVer)</PackageValidationBaselineVersion>
+    </PropertyGroup>
+  </Target>
```

Comment:
It's an anti-pattern to have a target in a props file.

Reply:
Fixed on #4927